This test fails on i686-linux. I couldn't come up with a regex that
could be used to disable it just on i686-linux, so we disable it
completely with this patch:

https://gitlab.freedesktop.org/mesa/mesa/-/issues/4091

diff --git a/src/util/meson.build b/src/util/meson.build
index 319b22d9bf7..93790c72675 100644
--- a/src/util/meson.build
+++ b/src/util/meson.build
@@ -344,7 +344,7 @@ if with_tests
     )
   endif
 
-  foreach t: ['bitset', 'register_allocate', 'u_debug_stack', 'u_qsort']
+  foreach t: ['bitset', 'register_allocate', 'u_qsort']
     test(
       t,
       executable(
